README.md

```diff
@@ -0,0 +1,55 @@
+# Math-magician-app
+
+"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+- Make simple calculations.
+- Read a random math-related quote.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+![screenshot](./calculator.png)
+
+## Built With
+
+- Javascript
+- HTML 5
+- CSS 3
+- React
+
+## Usage
+### Local installation
+```console
+$ git clone git@github.com:angelacuahutle/math-magicians-app.git
+$ cd math-magicians-app
+$ npm install 
+```
+
+### To run the project
+```console
+$ npm start 
+```
+
+## Live Demo
+
+[Live Demo Link](#)
```

Comment:
[Optional]
You can check [this link](https://gist.github.com/cobyism/4730490) for a simple guide for deploying react projects to GitHub pages